manifests/cinder-csi-plugin/csi-nodeplugin-cinderplugin.yaml

```diff
@@ -17,17 +17,21 @@ spec:
       serviceAccount: csi-nodeplugin
       hostNetwork: true
       containers:
-        - name: driver-registrar
-          image: quay.io/k8scsi/driver-registrar:v0.4.1
+        - name: node-driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.1
           args:
```

Comment:
why we are not adding cluster-driver-registrar here, it creates a cluster-level CSIDriver object , and says its recommended to easily discover the CSI drivers installed on their clusters.

Reply:
@ramineni yes we can deploy cluster-driver-registrar as well, but it necessary now as we are not using any of its features like pod info and skip attach, and one problem with current deployment yamls is we do not have single yaml for controller-plugin, I would prefer to add to cluster-driver-registrar while rearranging yamls for cinder csi plugin.

Reply:
@adisky ok, thanks. sounds good.